Se 13419 rtm infinite link loop hn (#216)
mulesoft/docs-runtime-manager: modules/ROOT/pages/servers-about.adoc

= Servers (Hybrid)
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]

> Runtime Manager provides an interface for managing on-premises Mule runtime engine instances (servers). You can manage servers individually or in server groups or clusters.

Reviewer: ```suggestion
Anypoint Runtime Manager provides an interface you use to manage your on-premises Mule runtime engine instances (servers). You can manage servers either individually or in server groups or clusters.
```

Servers communicate with Runtime Manager using Runtime Manager agent. After adding a server, use Runtime Manager to deploy applications to the Mule instance. You can also view information about the environment where you installed Mule.

> For each Mule that you want to manage, add a server to Runtime Manager. In environments with multiple Mule instances, use server groups and clusters for redundancy, high availability, and ease of management. 

Reviewer: ```suggestion
For each Mule instance that you want to manage, add a server to Runtime Manager. In environments with multiple Mule instances, use server groups and clusters for redundancy, high availability, and ease of management. 
```
